tests/test_ping_plugin.py

```
@@ -384,79 +367,225 @@ def test_handle_meshtastic_message_no_ping(self, mock_connect):
+        async def run_test():
+            result = await self.plugin.handle_meshtastic_message(
+                packet, "formatted_message", "TestNode", "TestMesh"
+            )
+            self.assertTrue(result)
+            mock_client.sendText.assert_called_once_with(text="Pong...", channelIndex=0)
 
         asyncio.run(run_test())
 
+    @patch("mmrelay.meshtastic_utils.connect_meshtastic")
+    @patch("asyncio.sleep")
+    def test_mimic_ping_in_sentence(self, mock_sleep, mock_connect):
```

Ruff (0.15.9)

[warning] Unused method argument: `mock_sleep`

(ARG002)